torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -1566,6 +1571,7 @@ void guardFusionGroup(
 
       // step a. handle fusion
       // remove inputs to fusion, and update check logic for fallback
+      auto profiled_ival_output = fusion->input(offset);
```

Comment:
I don't see `profiled_ival_output` used anywhere.

Reply:
I must have missed removing it.